ui/pages/send/send-content/send-content.component.js

```diff
@@ -103,15 +104,9 @@ export default class SendContent extends Component {
     const { t } = this.context;
     return (
       <div className="send__warning-container" data-testid="send-warning">
-        <ActionableMessage
-          type="danger"
-          useIcon
-          iconFillColor="var(--color-error-default)"
-          primaryActionV2={{
-            label: t('tooltipApproveButton'),
-            onClick: acknowledgeRecipientWarning,
-          }}
-          message={t('sendingToTokenContractWarning', [
+        <BannerAlert
+          severity={Severity.Danger}
+          description={t('sendingToTokenContractWarning', [
             <a
```

Comment:
I think we could use `ButtonLink` here(Will need to import `ButtonLinkSize` from component-library
```suggestion
            <ButtonLink
              key="contractWarningSupport"
              href={CONTRACT_ADDRESS_LINK}
              externalLink
              size={ButtonLinkSize.Inherit}
            >
              {t('learnMoreUpperCase')}
            </ButtonLink>,
```

We could also remove the styles for  className="send__warning-container__link" https://github.com/MetaMask/metamask-extension/blob/db064780d814477e71e3167ae34970e23ded09be/ui/pages/send/send.scss#L41-L43